README.md

```diff
@@ -16,11 +16,14 @@ You can choose between four different projects:
 - Sass
 - ESLint
 
-<img src="./src/images/StartScreen.png" alt="start screen" width="200">
-<img src="./src/images/Selection.png" alt="selection" width="200">
-<img src="./src/images/Dashboard.png" alt="dashboard" width="200">
-<img src="./src/images/LoadingState.png" alt="loading state" width="200">
-<img src="./src/images/ErrorState.png" alt="error state" width="200">
+It is also possible to create and delete your own issue. These appear in a separate list.
```

Comment:
little typo ;) In would recommend for e.g.: "...own issue, which appears in a separate list"